mozilla-release/other-licenses/nsis/Contrib/CliqzHelper/helper.cpp

```diff
@@ -130,3 +118,20 @@ std::wstring GetParentCommandString() {
   }
   return cmd;
 }
+
+std::wstring GetProcessFilename(ProcessId pid) {
+  std::wstring cmd;
+  if (pid != 0) {
+    TCHAR filename[MAX_PATH];
+
+    HANDLE processHandle = OpenProcess(
+        PROCESS_QUERY_INFORMATION | PROCESS_VM_READ, FALSE, pid);
+    if (processHandle != NULL) {
+      if (GetModuleFileNameEx(processHandle, NULL, filename, MAX_PATH)) {
+        cmd = filename;
+      }
+      CloseHandle(processHandle);

```

Comment:
This is not "mozilla based" project, it don't have any dependencies with mozilla source code.